tool/speed.cc

```diff
@@ -673,7 +689,11 @@ static bool SpeedHash(const EVP_MD *md, const std::string &name,
 
 static bool SpeedHmacChunk(const EVP_MD *md, std::string name,
                            size_t chunk_len) {
+#if defined(OPENSSL_1_0_BENCHMARK)
+  std::unique_ptr<HMAC_CTX> ctx = std::unique_ptr<HMAC_CTX>(new HMAC_CTX);
```

Comment:
I theory I think you are supposed to call `HMAC_CTX_init()` on the object. But in practice, meh, not sure it makes a difference.